one_fm/api/tasks.py

```diff
@@ -167,23 +167,6 @@ def notify_checkin_checkout_final_reminder(recipients,log_type):
 	elif log_type == "OUT":
 		send_notification(title, checkout_subject, checkout_message, notification_category, user_id_list)
 
-def insert_Contact():

```

Comment:
Is there any usage for this method

Reply:
No, This function isn't used anywhere. I was written by me to do some data entry.